src/scales/quantitativeScale.ts

```diff
@@ -115,8 +114,8 @@ module Plottable {
       }
     }
 
-    public _defaultExtent(): any[] {
-      return [0, 1];
+    public _defaultExtent(): D[] {
+      throw Error("The quantitative scale itself does not have a default extent");
```

Comment:
This is a good point. Now the problem is: 
1. We want to remove `any` everywhere it is possible. 
2. There is no default extent when there is no concrete type.

Also, a sign that there is a problem is the underscore in the name. I suggest we remove this method altogether. It is only used in the `domainer.ts`